src/test/java/pl/karolpat/service/VehicleServiceTest.java

```diff
@@ -0,0 +1,116 @@
+package pl.karolpat.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import pl.karolpat.entity.User;
+import pl.karolpat.entity.Vehicle;
+import pl.karolpat.repository.VehicleRepo;
+import pl.karolpat.serviceImpl.VehicleServiceImpl;
+
+@RunWith(SpringRunner.class)
+public class VehicleServiceTest {
+
+	private VehicleRepo vehicleRepo;
+	private VehicleService vehicleService;
+	Vehicle vehicle;
```

Comment:
czemu tu nie `private`?

Reply:
Właściwie, to nie potrafię odpowiedzieć na to pytanie. Jakiś głupi zwyczaj, rzeczywiście wygląda to niespójnie. Dzięki za zwrócenie uwagi.